HISTORY.md

```diff
@@ -5,6 +5,18 @@
 ### Breaking Changes
 
 + There is now a soft limit of 2 XRP on the fee specified in transaction instructions. Attempting to set a high fee will cause a `ValidationError` to be thrown. This error can be bypassed by setting `allowHighFee: true`.
++ The value returned by `getFee()` is capped at 1 XRP. You can ignore this cap by setting `maximum` to `null`.
++ When using `prepare*` methods, the `maxFee` parameter now has a default value of `'2'`.
+
+#### Overview of fee limits
+
+Most users of ripple-lib do not need to make any code changes to accommodate the new soft limits on fees. These limits are designed to protect against the most severe cases where an unintentionally high fee may be used.
+
++ When having ripple-lib provide the fee with a `prepare*` method, a default `maxFee` of `'2'` applies. You can prepare more economical transactions by setting `maxFee` lower, or support high-priority transactions by setting `maxFee` higher.
++ To set a specific fee higher than 2 XRP, set `allowHighFee: true`. Such a high fee is not typical, but ripple-lib permits it with this override.
++ When using `sign` with a Fee higher than 2 XRP, a `ValidationError` is thrown unless `allowHighFee` is set.
```

Comment:
This (unlikely) scenario seems unintuitive:
1. call `prepare*` with `maxFee` > 2 XRP and `allowHighFee` not set
2. calculated fee is > 2 XRP (and possibly capped at `maxFee`)
3. call to `sign` using fee returned by `prepare*` fails without setting `allowHighFee`

Reply:
Good point. So when someone calls `prepare*` with `maxFee` > 2 XRP, we should require `allowHighFee`?

Reply:
What if `maxFee` were an optional RippleAPI constructor [parameter](https://github.com/ripple/ripple-lib/blob/develop/docs/index.md#parameters) instead of an optional [transaction instruction](https://github.com/ripple/ripple-lib/blob/develop/docs/index.md#transaction-instructions) (similar to `feeCushion`)?
It could default to 2 XRP, and no calculated or specified fee would be allowed to exceed it. (We wouldn't have `allowHighFee`.)

Reply:
This is a good idea. I'll push a proposed change for this shortly.